.travis.yml

```diff
@@ -0,0 +1,35 @@
+---
+sudo: required
+dist: trusty
+language: node_js
+
+node_js:
+  - 9
+python:
+  - 3.6.x
+
+addons:
+  apt:
+    packages:
+      - imagemagick
+      - graphicsmagick
+
+install:
+  - wget https://github.com/gohugoio/hugo/releases/download/v0.31.1/hugo_0.31.1_Linux-64bit.deb
+  - sudo dpkg -i hugo*.deb
+  - npm install
+  - npm install --global gulp-cli
```

Comment:
Does [Pygments](https://github.com/nats-io/nats-site#install-pygments) need to be installed?

Reply:
Pygments doesn't even install on travis for some weird error. And current pushes of the site have not used it.

Reply:
OK.